var/spack/repos/builtin/packages/petsc/package.py

```diff
@@ -16,14 +16,21 @@ class Petsc(Package):
     """
 
     homepage = "http://www.mcs.anl.gov/petsc/index.html"
-    url      = "http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-3.5.3.tar.gz"
```

Comment:
You should keep the `url` because it's used by `spack versions` and `spack checksum`. You can update it to the new URL format if you want.

Reply:
updated